Gruntfile.js

```diff
@@ -255,100 +180,83 @@ module.exports = function (grunt) {
 				]
 			}
 		}
+	},
+	{
+		istanbulIgnoreNext: '/* istanbul ignore next */',
+		staticTestFiles: 'tests/**/*.{html,css}',
+		distDirectory: 'dist',
+		testDirectory: 'test',
+		srcDirectory: 'src',
+		targetDirectory: '<%= devDirectory %>',
+		siteDirectory: '.'
```

Comment:
When running `grunt` I get an error:

```
Warning: An error occurred while processing a template (siteDirectory is not defined). Use --force to continue.
```

It doesn't look like there is a 3rd parameter in grunt-dojo2?